### DISCUSSION TITLE
I built my personal website using fiber-go, elem-go, htmx and tailwind!

### DISCUSSION THREAD

**Discussion by dreth** [chasefleming/elem-go#124] [Show and tell]
Here's the code: https://github.com/dreth/dac.ac-go

And the URL: https://dac.ac

Really liking this stack (:

My code is far from 'good', as I'm just starting out with go, but I quite like the development experience so far and using elem-go, htmx and tailwind lets me write basically no html, css or js.

**Comment by chasefleming** [comment]
This is awesome to see @dreth ! Thanks for sharing this.
